docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,447 @@
+
+Depending on the type of sequencing that was done, the `read_groups` section of the payload will need to be adjusted. The following are examples of correctly formatted `read_groups` that meet [the rules](/docs/submission/molecular-data-prep#file-and-data-validation-rules) for paired-end vs single-end sequencing:
+
+<!---  Tabs start here -->
+
+<Tabs
+  defaultValue="BAM-paired"
+  values={[
+    { label: 'BAM (paired)', value: 'BAM-paired', },
+    { label: 'BAM (single)', value: 'BAM-single', },
+    { label: 'FASTQ (paired)', value: 'FASTQ-paired', },
+    { label: 'FASTQ (single)', value: 'FASTQ-single', },
+  ]
+}>
+<TabItem value="BAM-paired">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+  	"read_group_id_in_bam": "C0HVY.2",
+  	"submitter_read_group_id": "C0HVY.2",
+  	"platform_unit": "74_8a",
+  	"is_paired_end": true,
+  	"file_r1": "test_rg3.bam",
+  	"file_r2": "test_rg4.bam",
+  	"read_length_r1": 150,
+  	"read_length_r2": 16,
+  	"insert_size": 232,
+  	"sample_barcode": null,
+  	"library_name": "Pond-147579"
+  }
+```
+
+</TabItem>
+<TabItem value="BAM-single">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": "C0HVY.2", 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8a",
+    "is_paired_end": true,
+    "file_r1": "test_rg55.bam",
+    "file_r2": null,
+    "read_length_r1": 150,
+    "read_length_r2": null,
+    "insert_size": null,
+    "sample_barcode": null,
+    "library_name": "Pond-147579"
+  }
+```
+
+</TabItem>
+<TabItem value="FASTQ-paired">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": null, 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8b",
+    "is_paired_end": true,
+    "file_r1": "test_rg23.fq",
+    "file_r2": "test_rg24.fq",
+    "read_length_r1": 150,
+    "read_length_r2": 125, 
+    "insert_size": 298,
+    "sample_barcode": null,
+    "library_name": "Pond-147580"
+  }
+```
+
+</TabItem>
+<TabItem value="FASTQ-single">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": null, 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8b",
+    "is_paired_end": false,
+    "file_r1": "test_rg5.fq",
+    "file_r2": null,
+    "read_length_r1": 150,
+    "read_length_r2": null, 
+    "insert_size": 298,
+    "sample_barcode": null,
+    "library_name": "Pond-147580"
+  }
+```
+
+</TabItem>
+</Tabs>
+
+<!---  Tabs end here -->
+### File and Data Validation Rules
+
+Sequencing data of both `BAM` and `FASTQ` type files is accepted, with the exception of interleaved FASTQ files.  Metadata payloads like the example above must follow these data validations:
+
+1. Compression of FASTQ files is **required**; both _gzip_ (suffix .fq.gz or .fastq.gz) or _bz2_ (suffix .fq.bz2 or .fastq.bz2) are supported.
+1. For both FASTQ and BAM submission, all files in the `files` section must be unique in the payload.
```

Comment:
what does "must be unique in the payload" mean? 
like should it say something likes
`must be uniquely identified in the payload`?

Reply:
means no duplicate files are allowed; you cannot upload the same file twice